livre/eq/EventHandler.cpp

```diff
@@ -274,6 +274,13 @@ template< class C > void EventHandler< C >::init()
 }
 
 template< class C >
+bool EventHandler< C >::handleEvent( eq::EventType, const eq::Event& )
+{
+    _impl->config.postRedraw();
```

Comment:
even those like CLOSE / HIDE / EXIT ? I would probably have listed made a white-list, but as long as this has no adverse side effect then I guess it's fine.